x-pack/plugins/maps/ui_tests/README.md

```diff
@@ -0,0 +1,20 @@
+## How to run tests
+First start the servers with
+
+```bash
+// ESS
+node scripts/scout_start_servers.js --stateful
+// Serverless
+node scripts/scout_start_servers.js --serverless=es
```

Comment:
These commands are outdated (this looks like copy/paste - not sure where these are coming from but we need to update the source as well). Should rather be:
```suggestion
// ESS
node scripts/scout.js start-server --stateful

// Serverless
node scripts/scout.js start-server --serverless=[es|oblt|security]
```

Reply:
Yeah this is not ready for review yet.  Not even close lol